File: dataset/human/java/file_2793.java
import java.util.*;
public class Main {
  public static void main(String[] args){
    Scanner sc = new Scanner(System.in);

    String s = sc.next();
    String t = sc.next();
    String ss[] = s.split("");
    String tt[] = t.split("");
    Arrays.sort(ss);
    Arrays.sort(tt);
    s = String.join("",ss);
    t = String.join("",tt);
    int n = Math.min(s.length(),t.length());
    String ans = "No";
    int c = 0;
    for(int i = 0 ; i < n ; ++i){
      if((int)s.charAt(i)<(int)t.charAt(t.length()-i-1)){
        ans = "Yes";
        break;
      }else if((int)s.charAt(i)==(int)t.charAt(t.length()-i-1)){
        c++;
      }
    }
    
    if(c==n && s.length()<t.length()) ans ="Yes";
    
    System.out.println(ans);

  }
}

File: dataset/human/c/file_2793.c
#include<stdio.h>

int min(int x,int y){
	if(x<y)return x;
	return y;
}

int main(){
	int K,S,ans=0;
	scanf("%d%d",&K,&S);
	for(int x=0;x<=min(K,S);x++)
		for(int y=0;y<=min(K,S);y++)if(S-x-y>=0 && S-x-y<=K)ans++;
			
	printf("%d\n",ans);
	return 0;
}
